stdlib/2and3/decimal.pyi

```diff
@@ -126,6 +126,12 @@ class Decimal(object):
         def __ceil__(self) -> int: ...
     else:
         def __long__(self) -> long: ...
+    @overload
+    def __round__(self) -> int: ...
+    @overload
+    def __round__(self, ndigits: None) -> int: ...
+    @overload
+    def __round__(self, ndigits: int) -> Decimal: ...
```

Comment:
These overloads should replace the existing `__round__` method inside the version guard block.